scripts/core/ui/views/sd-datepicker.html

```diff
@@ -1,6 +1,6 @@
 <div class="datepicker-input">
      <input type="text"
-         class="input-small"
+         class=""
```

Comment:
Remove empty attribute

Reply:
UI framework is updated, it should be fine once the PR is merged. Class removed